components/widgets/admin_console/formatted_admin_header.tsx

```diff
@@ -4,10 +4,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import AdminHeader from 'components/widgets/admin_console/admin_header.jsx';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
-const FormattedAdminHeader = (props) => (
+import AdminHeader from './admin_header';
+
+type Props = {
+    id: string,
+    defaultMessage: string,
+    values?: any,
+};
+
+const FormattedAdminHeader = (props:Props) => (
```

Comment:
React.FC<Props>

Reply:
Also, this  `(props:Props)` shouldn't pass lint (space needed after semi-colon). We should integrate eslint-typescript. I can help with that

Reply:
I going to add the typescript mod for eslint